README.md

```diff
@@ -0,0 +1,47 @@
+# secret-script
+Proof-of-concept python script for sychronization between kubernetes secret and gcloud secret manager.
+
+## Prerequisites
+Create a gke cluster.
+
+[Create service account for app](https://cloud.google.com/docs/authentication/production#command-line)
```

Comment:
What permissions does the service account need? When I ran this with a freshly created service account I ran into a bunch of errors.

Reply:
Added to README.

Will need container.secrets.* and container.clusterViewer permissions.